endtoend/minimal_e2e_test.go

```diff
@@ -19,6 +19,7 @@ func TestEndToEnd_MinimalConfig(t *testing.T) {
 
 // Run minimal e2e config with the current release validator against latest beacon node.
 func TestEndToEnd_MinimalConfig_ValidatorAtCurrentRelease(t *testing.T) {
+	t.Skip("Skipping for Altair hard fork as there are breaking changes")
```

Comment:
This puts us at a high risk for introducing breaking or non-backwards compatible changes. 
Is there anything else we can do? Skip certain evaluators for prior release?